runtime/tests/port/si.c

```diff
@@ -2199,13 +2207,13 @@ j9sysinfo_test_get_l1dcache_line_size(struct J9PortLibrary *portLibrary)
 					l1DcacheSize, rc);
 		}
 	}
-#else /* !(defined(J9ARM) || defined(RISCV64)) */
+#else /*!(defined(J9ARM))*/
```

Comment:
Please update the comment to match the `#if` condition and restore the spaces (here and on the `#endif`).
```
#else /* !defined(J9ARM) */
```

Reply:
Fixed.